crates/biome_diagnostics/Cargo.toml

```
@@ -35,6 +35,7 @@ biome_rowan                  = { workspace = true }
 biome_text_edit              = { workspace = true }
 biome_text_size              = { workspace = true }
 bpaf                         = { workspace = true }
+camino                       = { workspace = true }
```

```suggestion
camino                       = { workspace = true, optional = true }
```

A feature `camino` seems to miss in this Cargo file.